src/PIL/AvifImagePlugin.py

```diff
@@ -0,0 +1,294 @@
+        self.is_animated = self.n_frames > 1
+        self._mode = mode
+
+        if icc:
+            self.info["icc_profile"] = icc
+        if exif:
+            self.info["exif"] = exif
+        if xmp:
+            self.info["xmp"] = xmp
+
+    def seek(self, frame: int) -> None:
+        if not self._seek_check(frame):
+            return
+
+        self.__frame = frame
+
+    def load(self) -> Image.core.PixelAccess | None:
+        if self.__loaded != self.__frame:
+            # We need to load the image data for this frame
+            data, timescale, tsp_in_ts, dur_in_ts = self._decoder.get_frame(
+                self.__frame
+            )
+            self.info["timestamp"] = round(1000 * (tsp_in_ts / timescale))
+            self.info["duration"] = round(1000 * (dur_in_ts / timescale))
+            self.__loaded = self.__frame
+
+            # Set tile
+            if self.fp and self._exclusive_fp:
+                self.fp.close()
+            self.fp = BytesIO(data)
+            self.tile = [ImageFile._Tile("raw", (0, 0) + self.size, 0, self.mode)]
+
+        return super().load()
+
+    def tell(self) -> int:
+        return self.__frame
+
+
+def _save_all(im: Image.Image, fp: IO[bytes], filename: str | bytes) -> None:
+    _save(im, fp, filename, save_all=True)
+
+
+def _save(
+    im: Image.Image, fp: IO[bytes], filename: str | bytes, save_all: bool = False
+) -> None:
+    info = im.encoderinfo.copy()
+    if save_all:
+        append_images = list(info.get("append_images", []))
+    else:
+        append_images = []
+
+    total = 0
+    for ims in [im] + append_images:
+        total += getattr(ims, "n_frames", 1)
+
+    is_single_frame = total == 1
+
+    qmin = info.get("qmin", -1)
+    qmax = info.get("qmax", -1)
+    quality = info.get("quality", 75)
+    if not isinstance(quality, int) or quality < 0 or quality > 100:
+        msg = "Invalid quality setting"
+        raise ValueError(msg)
+
+    duration = info.get("duration", 0)
+    subsampling = info.get("subsampling", "4:2:0")
+    speed = info.get("speed", 6)
+    max_threads = info.get("max_threads", _get_default_max_threads())
+    codec = info.get("codec", "auto")
+    if codec != "auto" and not _avif.encoder_codec_available(codec):
+        msg = "Invalid saving codec"
+        raise ValueError(msg)
+    range_ = info.get("range", "full")
+    tile_rows_log2 = info.get("tile_rows", 0)
+    tile_cols_log2 = info.get("tile_cols", 0)
+    alpha_premultiplied = bool(info.get("alpha_premultiplied", False))
+    autotiling = bool(info.get("autotiling", tile_rows_log2 == tile_cols_log2 == 0))
+
+    icc_profile = info.get("icc_profile", im.info.get("icc_profile"))
+    exif = info.get("exif", im.info.get("exif"))
+    if isinstance(exif, Image.Exif):
+        exif = exif.tobytes()
+
+    exif_orientation = 0
+    if exif:
+        exif_data = Image.Exif()
+        try:
+            exif_data.load(exif)
+        except SyntaxError:
+            pass
```

Comment:
Could you talk about why invalid EXIF data is something to be ignored, and not something that should stop the save operation?

Reply:
I've created https://github.com/fdintino/Pillow/pull/8 to not catch the error.